.bem/templates/js.js

```diff
@@ -12,19 +11,20 @@ function toObjectValue(x) {
 
 module.exports = function (entity, naming) {
     return [
-        "import {decl} from 'bem-react-core';",
+        "import { decl } from 'bem-react-core';",
         "",
         "export default " +
-        (entity.modName ?
+        (Object(entity.mod).name ?
```

Comment:
In current version of `@bem/entity-name` API was changed to `mod: { name, val }`.
Old version with `modName` is still supported but generates warnings.